### DISCUSSION TITLE
Is there a place where I can find an up-to-date full list of attributes supported by Prisma?

### DISCUSSION THREAD

**Discussion by ymc9** [prisma/prisma#18287] [Q&A] [answered]
Ideally with those used by experimental features. Thanks!

**Comment by janpio** [comment, marked as answer]
Usually that would be https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#attributes

Is that what you meant by "attributes" or did you mean it in a wider sense?

**Reply by ymc9** [reply]
Thanks! I'm aware of this documentation. It just seems that attributes for experimental features are not listed here, e.g., `@schema`. I'm working on a project that wishes to fully cover (and also keep up with) all Prisma attributes 😄.

**Reply by janpio** [reply]
Fair point. That is more an oversight than intentional. I'll raise this internally.

**Reply by nikolasburk** [reply]
I created a docs issue for this: https://github.com/prisma/docs/issues/4580 🙏
